source/install/troubleshooting.rst

```diff
@@ -19,9 +19,41 @@ General Troubleshooting
 -----------------------
 Some of these suggestions can be done directly, and others may need consultation from your network administrator.
 
-- Take a look at the logs (``mattermost.log`` and NGINX logs) for errors.
-- You can also search the error messages online - existing solutions can often be applied.
-- Open **System Console > General > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space.
+Review Mattermost Logs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can access logs for Mattermost and use them for troubleshooting. These steps assume that you have `System Admin permissions <https://docs.mattermost.com/help/getting-started/managing-members.html#system-admin>`_. 
+
+**Mattermost Server**
+
+#. Ensure that log files are being created: Navigate to **System Console > Environment > Logging**, confirm that **Output logs to file** is set to **true**.
+#. You can obtain the path for the log files in **System Console > Environment > Logging > File Log Directory**.
```

Comment:
This isn't rendering correctly (one is a number and one is just the #). For a bulleted list, try using "-", that could resolve the issue. :)